templates/tfengine/components/cicd/triggers/plan/main.tf

```diff
@@ -0,0 +1,118 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_cloudbuild_trigger" "plan_env" {
+  count       = var.skip ? 0 : 1
+  disabled    = var.run_on_push
+  provider    = google-beta
+  project     = var.project_id
+  name        = "tf-plan-${var.env}"
+  description = "Terraform plan job triggered on push event."
+
+  included_files = [
+    "${var.terraform_root_prefix}**",
+  ]
+
+  {{if has $ "github" -}}
+  github {
+    owner = var.github.owner
+    name  = var.github.name
+    pull_request {
+      branch = "^${var.branch_name}$"
+    }
+  }
+  {{- else if has $ "cloud_source_repository" -}}
+  trigger_template {
+    repo_name   = var.cloud_source_repository.name
+    branch_name = "^${var.branch_name}$"
+  }
+  {{- end}}
+
+  filename = "${var.terraform_root_prefix}cicd/configs/tf-plan.yaml"
+
+  substitutions = {
+    _TERRAFORM_ROOT = var.terraform_root
+    _MANAGED_DIRS = var.managed_dirs
+  }
+
+  depends_on = [
+    google_project_service.services,
```

Comment:
`google_project_service.services` does not exist in this submodule anymore. This will error out.

Reply:
same in other places

Reply:
You're right. I removed them and pass the `google_source_repository.configs` conditioned to the depends_on at `cicd/main.tf`